auth/cache/redis_test.go

```
@@ -0,0 +1,241 @@
+		}
+	}
+	if entry.UpdatedAt.IsZero() {
+		t.Error("GetFUPEntry() updatedAt is zero, want the increment timestamp")
+	}
+}
+
+func TestRedisCacheDriver_IncrementFUPEntry_TTL(t *testing.T) {
+	driver, server := newTestRedisDriver(t)
+	// an entry stored by an older version of the package never expires
+	storeFUPEntry(t, server, "key", stableNow(t), nil)
+	if ttl := server.TTL("test:fup_key"); 0 != ttl {
```

golangci-lint (2.12.2)

[error] ST1017: don't use Yoda conditions

(staticcheck)